packages/@react-stately/calendar/src/useRangeCalendarState.ts

```diff
@@ -33,15 +33,26 @@ export interface RangeCalendarStateOptions<T extends DateValue = DateValue> exte
    * The amount of days that will be displayed at once. This affects how pagination works.
    * @default {months: 1}
    */
-  visibleDuration?: DateDuration
+  visibleDuration?: DateDuration,
+  /** Determines the alignment of the visible months on initial render based on the current selection or current date if there is no selection. */
```

Comment:
Realised this probably also affects `focusedValue` so it's probably not just the initial render, it's probably anytime that that prop changes as well?
Could you add something to control that value? Maybe a second story based on https://react-spectrum.adobe.com/react-aria/Calendar.html#controlling-the-focused-date ? Just so we know what the interaction is

Reply:
I think maybe not? Otherwise if some is controlling the focus, it could cause too much "movement" just moving one month to the next.

The reset is where I get tripped up because then I think it should...